src/images/gorilla_icon.svg

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This comment is for me to remember that we should look into a way that we can optimize our `svg` assets, for now this is great 👍